dist/rpm/openQA.spec

```diff
@@ -769,6 +786,9 @@ fi
 %{_bindir}/openqa-load-templates
 %{_bindir}/openqa-clone-custom-git-refspec
 %{_bindir}/openqa-validate-yaml
+%dir %{_datadir}/openqa/completion
```

Comment:
That's not going to work. People would still have to manually add these files into their rc files.
I showed you how git-subrepo is doing it.
Just look at `rpm -ql git-subrepo-zsh-completion git-subrepo-bash-completion`.

What we need in the end is this (and with those exact filenames):
```
/usr/share/zsh/site-functions/_openqa-cli
/usr/share/bash-completion/completions/openqa-cli
```

Reply:
Also you are listing those files as part of the client package (see `%files client` above), but it should go to `%files client-bash-completion` and `%files client-zsh-completion` respectively.

Reply:
As always I noticed this a bit late. and once I realise that the path should be different. I try to make it work as you suggest. I will let you know